packages/cli/src/lib/cmds/build.ts

```diff
@@ -143,6 +159,20 @@ class Build {
     const manifestFile = this.args.manifest || path.join(process.cwd(), TWA_MANIFEST_FILE_NAME);
     const twaManifest = await TwaManifest.fromFile(manifestFile);
 
+    const manifestCheck = await this.checkManifest(manifestFile);
+
+    if (manifestCheck) {
+      const applyChanges = await this.prompt.promptConfirm(messages.promptUpdateProject, true);
+      if (applyChanges) {
+        const updated = await updateProject(false, '', this.prompt, '', manifestFile);
```

Comment:
Shouldn't the `appVersionName` parameter here be not null? (Also I'm a bit dubious about the directory being null as well.)

Reply:
I changed the directory so it'll use `args.directory`. I'm not sure what to do about `appVersionName` though, because this update is happening when they run `build` so the user won't be passing in `appVersionName`.

Reply:
OK, so it looks like the code will continue to work if `appVersionName` is empty - the `updateVersions` method should deal with it.

I'm not too fond of passing empty strings around - if anything we should make `appVersionName` nullable and pass null around to make it really clear. Also, the documentation for `updateProject` should make it clear that `appVersionName` can be null, and if so a default will be used (if needed).

Do you mind changing appVersion name to a `string?` and adding such documentation to `updateProject`?

Reply:
I made `appVersionName` nullable and also updated documentation for `updateProject`. Let me know if this is okay or if it makes more sense to make `appVersionName` an optional parameter instead. Thanks!